docs/getting_started_with_jax_for_AI.md

```diff
@@ -82,8 +80,8 @@ for i, ax in enumerate(axes.flat):
 
 +++ {"id": "Z3l45KgtfUUo"}
 
-Let's split these into a training and testing set, and convert these splits into JAX arrays which will be ready to feed into our model.
-We'll make use of the `jax.numpy` module, which provides a familiar NumPy-style API around JAX operations:
+Next, split the dataset into a training and testing set, and convert these splits into [`jax.Array`s]https://jax.readthedocs.io/en/latest/key-concepts.html#jax-arrays-jax-array) before you feed them into the model.
+You’ll use the `jax.numpy` module, which provides a familiar NumPy-style API around JAX operations:
```

Comment:
Out of curiosity, why the switch from "we" to "you"?

Reply:
@jakevdp Good question! Second person - guidelines https://developers.google.com/style/person

> "In general, address the reader of your documents using the second person instead of the first person: use you or your instead of we, our, or us. Assume that the reader is the person who's doing the tasks that you're documenting. Use the word user only to refer to the user of the software that your reader is developing."

Reply:
Interesting – I've used "we" in all my tech writing for years. "you" sounds strange to me...

Reply:
"We" is good. Not rules these are https://developers.google.com/style/person :+1: 

And addressing the reader with "you" also makes sense, from the user's point of view. Maybe this will also translate better if we have translation in future?